config.js

```diff
@@ -94,7 +94,7 @@ config.CLOUD_ALERT_GRACE_TIME = 3 * 60 * 1000; // grace period before dispatchin
 config.AGENT_RESPONSE_TIMEOUT = 1 * 60 * 1000;
 config.AGENT_TEST_CONNECTION_TIMEOUT = 1 * 60 * 1000;
 config.STORE_PERF_TEST_INTERVAL = 60 * 60 * 1000; // perform test_store_perf every 1 hour
-config.CLOUD_MAX_ALLOWED_IO_TEST_ERRORS = 3;
+config.CLOUD_MAX_ALLOWED_IO_TEST_ERRORS = 4;
```

Comment:
@liranmauda I removed the env, because we can edit the value by using the prefix `CONFIG_JS_`
https://github.com/noobaa/noobaa-core/blob/26ee66b74f3163fa791e107acdec07f207769606/config.js#L863